README.md

```diff
@@ -32,7 +32,7 @@ cargo install bpf-linker
 
 On Debian based distributions you need to install the `llvm-16-dev` and
 `libclang-16-dev` packages. If your distro doesn't have them you can get them
-from the official LLVM repo at https://apt.llvm.org.
+from the official LLVM repo at https://apt.llvm.org. You need to install libpolly-16-dev too if you use this repo.
```

Comment:
why is this not in the first sentence that mentions llvm-16-dev and libclang-16-dev?

also please keep the column width consistent.

Reply:
my impression was it's only needed for packages from apt.llvm.org. Double checked with the recent Ubuntu - it's needed there too. Will fix my changes shortly